config/routes.rb

```diff
@@ -278,6 +279,21 @@
     resources :items, controller: 'dataset_items', defaults: {format: 'json'}
   end
 
+
+  # studies, questions, responses
+  put 'responses/:id', to: 'errors#method_not_allowed_error'
+  put '/studies/:study_id/responses/:id', to: 'errors#method_not_allowed'
+  match 'studies/filter' => 'studies#filter', via: [:get, :post], defaults: {format: 'json'}
+  match 'questions/filter' => 'questions#filter', via: [:get, :post], defaults: {format: 'json'}
+  match 'responses/filter' => 'responses#filter', via: [:get, :post], defaults: {format: 'json'}
+  resources :studies, defaults: {format: 'json'}
+  resources :questions, defaults: {format: 'json'}
+  resources :responses, except: :update, defaults: {format: 'json'}
+  get '/studies/:study_id/questions' => 'questions#index', defaults: {format: 'json'}
+  get '/studies/:study_id/responses' => 'responses#index', defaults: {format: 'json'}
+  post '/studies/:study_id/questions/:question_id/responses' => 'responses#create', defaults: {format: 'json'}
```

Comment:
please redefine this whole block of routes in terms of `resources` definitions, as is the style in the rest of this file